velox/exec/HashTable.cpp

```diff
@@ -798,7 +798,8 @@ void syncWorkItems(
     bool log = false) {
   // All items must be synced also in case of error because the items
   // hold references to the table and rows which could be destructed
-  // if unwinding the stack did ont pause to sync.
+  // if unwinding the stack did not pause to sync.
+  TestValue::adjust("facebook::velox::exec::HashTable::syncWorkItems", nullptr);
```

Comment:
Can we add testvalue in AsyncSource::move so it might be used in the other case? And you just need to inject the testvalue once on the first call to move? thanks!